packages/bee-q/src/components/breadcrumb/bq-breadcrumb.tsx

```diff
@@ -0,0 +1,131 @@
+import { h, Component, Element, Listen, Event, EventEmitter, Host, Prop } from '@stencil/core';
+
+import { isHTMLElement } from '../../shared/utils';
+
+/**
+ * @part navigation - The `nav` tag that loads the breadcrumb items
+ */
+@Component({
+  tag: 'bq-breadcrumb',
+  styleUrl: './scss/bq-breadcrumb.scss',
+  shadow: true,
+})
+export class BqBreadcrumb {
+  // Own Properties
+  // ====================
+
+  private spanElem: HTMLElement;
+
+  // Reference to host HTML element
+  // ===================================
+
+  @Element() el!: HTMLBqBreadcrumbElement;
+
+  // State() variables
+  // Inlined decorator, alphabetical order
+  // =======================================
+
+  // Public Property API
+  // ========================
+
+  /** The `aria-label` attribute for `<nav>` */
+  @Prop({ reflect: true }) ariaLabel: string = 'breadcrumbs';
```

Comment:
```suggestion
  /** The `aria-label` attribute to describe the type of navigation */
  @Prop({ reflect: true }) ariaLabel: string;
```

Reply:
Shouldn't we use a default `aria-label` value for the `nav`? In case the user will not add any value then we have `breadcrumb` as default.

Reply:
Then use `Breadcrumbs` (with Capitalize letter) please 🙂